test/fit_tests/config/stack100.json

```diff
@@ -0,0 +1,27 @@
+{
+  "100": {
+    "bmc": "stack100-bmc.admin",
+    "control": "stack100-control.admin",
+    "data": "stack100-data.admin",
+    "hyper": "stack100-esxi.admin",
+    "ora": "stack100-ora.admin",
+    "ovamac": "00:50:56:00:11:00",
+    "pdu": "192.168.1.254",
+    "type": "esxi",
+    "nodes": [
+      {"bmcmac": "00:1e:67:d5:e1:96", "nodeid": "", "sku": "Rinjin KP"},
```

Comment:
As I know, the server name listed in https://eos2git.cec.lab.emc.com/OnRack/emc-hardware-support could be published to public, there is a public  name such as 'platform-r' for Rxxjxx, so please confirm with Rod or someone for this